packages/languages/src/node/language-server-contribution.ts

```diff
@@ -112,8 +112,8 @@ export abstract class BaseLanguageServerContribution implements LanguageServerCo
             rawProcess.onError((error: ProcessErrorEvent) => {
                 this.onDidFailSpawnProcess(error);
                 if (error.code === 'ENOENT') {
-                    const guess = command.split('\S').shift();
-                    if (guess) {
+                    const guess = command.split(/\s+/).shift();
+                    if (typeof guess === 'string') {
```

Comment:
Why was it necessary to change this line?

Reply:
I wanted to avoid logging `[Object object]` in any case.

Reply:
Hmm, string.split(..).shift() can return something else than a string?

Reply:
Ah, maybe because the typing claims it can return undefined too (if the array is empty)?